misc/run-phoenix-release-gpu.sh

```diff
@@ -17,5 +17,4 @@ echo $(pwd)
 nvidia-smi
 echo $(nproc)
 
-
-./mfc.sh test -b mpirun -a --gpu
+./mfc.sh test -b mpirun -a --gpu
```

Comment:
Did you intend to include this change?

Reply:
@henryleberre, finally, same here. There should be no (intended) changes to this file other than the ones I followed from you